cdc/sink/producer/kafka/kafka.go

```diff
@@ -550,7 +604,9 @@ func newSaramaConfig(ctx context.Context, c *Config) (*sarama.Config, error) {
 	}
 	config.Version = version
 	// See: https://kafka.apache.org/documentation/#replication
-	// When one of the brokers in a Kafka cluster is down, the partition leaders in this broker is broken, Kafka will election a new partition leader and replication logs, this process will last from a few seconds to a few minutes. Kafka cluster will not provide a writing service in this process.
+	// When one of the brokers in a Kafka cluster is down, the partition leaders in this broker is broken,
+	// Kafka will election a new partition leader and replication logs, this process will last from a few seconds to a few minutes.
```

Comment:
Please keep line width within 80 chars.
  
```suggestion
	// When one of the brokers in a Kafka cluster is down, the partition leaders
	// in this broker is broken, Kafka will election a new partition leader and
	// replication logs, this process will last from a few seconds to a few
	// minutes.
```